dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/builders/TripleBuilder.java

```diff
@@ -0,0 +1,199 @@
+
+    /**
+     * Whether to enable push, default is false.
+     */
+    private Boolean enablePush;
+
+    /**
+     * Maximum concurrent streams.
+     */
+    private Integer maxConcurrentStreams;
+
+    /**
+     * Initial window size.
+     */
+    private Integer initialWindowSize;
+
+    /**
+     * Maximum frame size.
+     */
+    private Integer maxFrameSize;
+
+    /**
+     * Maximum header list size.
+     */
+    private Integer maxHeaderListSize;
+
+    /**
+     * Whether to pass through standard HTTP headers, default is false.
+     */
+    private Boolean passThroughStandardHttpHeaders;
+
+    public static TripleBuilder newBuilder() {
+        return new TripleBuilder();
+    }
+
+    public TripleBuilder maxBodySize(Integer maxBodySize) {
+        this.maxBodySize = maxBodySize;
+        return getThis();
+    }
+
+    public TripleBuilder maxResponseBodySize(Integer maxResponseBodySize) {
+        this.maxResponseBodySize = maxResponseBodySize;
+        return getThis();
+    }
+
+    public TripleBuilder maxChunkSize(Integer maxChunkSize) {
+        this.maxChunkSize = maxChunkSize;
+        return getThis();
+    }
+
+    public TripleBuilder maxHeaderSize(Integer maxHeaderSize) {
+        this.maxHeaderSize = maxHeaderSize;
+        return getThis();
+    }
+
+    public TripleBuilder maxInitialLineLength(Integer maxInitialLineLength) {
+        this.maxInitialLineLength = maxInitialLineLength;
+        return getThis();
+    }
+
+    public TripleBuilder initialBufferSize(Integer initialBufferSize) {
+        this.initialBufferSize = initialBufferSize;
+        return getThis();
+    }
+
+    public TripleBuilder headerTableSize(Integer headerTableSize) {
+        this.headerTableSize = headerTableSize;
+        return getThis();
+    }
+
+    public TripleBuilder enablePush(Boolean enablePush) {
+        this.enablePush = enablePush;
+        return getThis();
+    }
+
+    public TripleBuilder maxConcurrentStreams(Integer maxConcurrentStreams) {
+        this.maxConcurrentStreams = maxConcurrentStreams;
+        return getThis();
+    }
+
+    public TripleBuilder initialWindowSize(Integer initialWindowSize) {
+        this.initialWindowSize = initialWindowSize;
+        return getThis();
+    }
+
+    public TripleBuilder maxFrameSize(Integer maxFrameSize) {
+        this.maxFrameSize = maxFrameSize;
+        return getThis();
+    }
+
+    public TripleBuilder maxHeaderListSize(Integer maxHeaderListSize) {
+        this.maxHeaderListSize = maxHeaderListSize;
+        return getThis();
+    }
+
+    public TripleBuilder passThroughStandardHttpHeaders(Boolean passThroughStandardHttpHeaders) {
+        this.passThroughStandardHttpHeaders = passThroughStandardHttpHeaders;
+        return getThis();
+    }
+
+    protected TripleBuilder getThis() {
+        return this;
+    }
+
+    public TripleConfig build() {
+        TripleConfig triple = new TripleConfig();
+
+        triple.setMaxBodySize(maxBodySize);
```

Comment:
Need if condition,  otherwise, it will be overwritten as null

Reply:
Okay, I'll fix it.